### PR TITLE
Comment on github when an error occurs

### PR DESCRIPTION
**Pull request by Ryman**

### CODE REVIEW

File: lib/reina/app.rb [hunk 1]
```diff
@@ -166,6 +166,12 @@ def parallel?
       project[:parallel] != false
     end
 
+    def show_live_url?
+      whitelist = CONFIG[:apps_with_live_url]
+      return true if whitelist.nil?
```
Comment (oluosiname) [inline lib/reina/app.rb]: @Ryman I think maybe this should return false when whitelist is missing. I'm assuming whitelist is supposed to always be present though

Comment (Ryman) [reply lib/reina/app.rb]: Without a whitelist everything should show (which means our current configs/behavior works), if we require that whitelist exists then it breaks current deployment(s)

**Comment by Ryman** [comment]
> I was also wondering if there's any reason we have `DeploymentError` class inside of the controller.

No real reason other than that I generally scope things to their use, happy to move it to toplevel if it's a blocker?

File: lib/reina/app.rb (hunk 1 above)
Comment (oluosiname) [reply lib/reina/app.rb]: All clear then

**Pull request merged**